apps/store/src/blocks/FooterBlock.tsx

```diff
@@ -93,95 +93,86 @@ export const FooterBlock = ({ blok, onLocaleChange }: FooterBlockProps) => {
   const footerSections = filterByBlockType(blok.sections, FooterSection.blockName)
   return (
     <Wrapper>
-      <Content>
-        {footerSections.map((nestedBlok) => (
-          <FooterSection key={nestedBlok._uid} blok={nestedBlok} />
-        ))}
-
-        <Bottom>
-          <Disclaimer>
-            <Text color="textSecondary" size="sm">
-              © Hedvig 2023
-            </Text>
-          </Disclaimer>
-
-          <LocaleForm>
-            <StyledInputSelect
-              name={LocaleField.Country}
-              onChange={handleChangeCountry}
-              defaultValue={currentCountry.id}
-              options={countryList}
-            />
-            <StyledInputSelect
-              name={LocaleField.Language}
-              onChange={handleChangeLanguage}
-              defaultValue={currentLanguage}
-              options={languageList}
-            />
-          </LocaleForm>
-        </Bottom>
-      </Content>
+      {footerSections.map((nestedBlok) => (
+        <Column key={nestedBlok._uid}>
+          <FooterSection blok={nestedBlok} />
+        </Column>
+      ))}
+
+      <LocaleForm>
+        <StyledInputSelect
+          name={LocaleField.Country}
+          onChange={handleChangeCountry}
+          defaultValue={currentCountry.id}
+          options={countryList}
+        />
+        <StyledInputSelect
+          name={LocaleField.Language}
+          onChange={handleChangeLanguage}
+          defaultValue={currentLanguage}
+          options={languageList}
+        />
+      </LocaleForm>
+
+      <Disclaimer>
+        <Text color="textSecondary" size="sm">
+          © Hedvig 2023
+        </Text>
+      </Disclaimer>
     </Wrapper>
   )
 }
 FooterBlock.blockName = 'footer' as const
 
 const Wrapper = styled.footer({
-  width: '100%',
   backgroundColor: theme.colors.gray100,
   paddingInline: theme.space.md,
   paddingTop: theme.space.xxl,
 
   // Clear floating price calculator button
-  paddingBottom: theme.space[9],
-})
+  paddingBottom: theme.space[10],
 
-const Content = styled.div({
   display: 'grid',
-  gridTemplateColumns: 'repeat(2, 1fr)',
-  gridTemplateRows: 'repeat(2, auto)',
+  gridTemplateColumns: 'repeat(12, 1fr)',
   columnGap: theme.space.md,
   rowGap: theme.space.xxl,
+})
 
-  maxWidth: `calc(8 * 6.25rem + 7 * ${theme.space.md})`,
-  marginInline: 'auto',
+const Column = styled.div({
+  gridColumn: 'span 6',
 
-  [mq.md]: {
-    gridTemplateColumns: 'repeat(4, 1fr)',
+  [mq.md]: { gridColumn: 'span 3' },
+  [mq.xxl]: {
+    gridColumn: 'span 2',
+    ':first-child': { gridColumn: '3 / span 2' },
```

Comment:
nit: same result, but played around a bit before and I think it's slightly easier to understand when you just override `gridColumnStart` 
```suggestion
    gridColumn: 'auto / span 2',
    ':first-child': { gridColumnStart: '3' },
```